### DISCUSSION TITLE
Welcome to erg Discussions!

### DISCUSSION THREAD

**Discussion by mtshiba** [erg-lang/erg#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by justin0mcateer** [comment]
Looks awesome!

**Comment by mtshiba** [comment]
Thank you.
Erg is a project that has just begun.
If you would like, please share this project with your friends and colleagues.
